vrtool_docs/Preprocessing/Vakindeling.md

```diff
@@ -18,56 +18,34 @@ Dit bestand heeft de volgende kolommen die ingevul dmoeten worden:
 | in_analyse  	     | Verplicht 	| Vak meenemen in analyse of niet (TRUE/FALSE)                                                                                                                                                 	                                                                                                                                                                                |
 | van_dp      	     | Optioneel 	| Begrenzing dijkpaalnummers                                                                                                                                                                   	                                                                                                                                                                                |
 | tot_dp      	     | Optioneel 	| Begrenzing dijkpaalnummers                                                                                                                                                                   	                                                                                                                                                                                |
-| piping          	 | Optioneel 	| Moet overeen komen met kolom 'doorsnede' bij [Piping_default.csv](Piping.md). Hiermee wordt een link gemaakt tussen vak en doorsnede die gebruikt moet worden voor de piping invoer. Wanneer dit niet wordt ingevuld moet dit later worden afgeleid uit bijv een shapefile met uittredepunten. Daarvoor zijn specifieke workflows beschikbaar. 	                              |
-| overslag      	   | Optioneel 	| Moet overeen komen met kolom 'doorsnede' bij [HR_default.csv](Overtopping.md), zie afbeelding hieronder. Hiermee wordt een link gemaakt tussen vak en doorsnede die gebruikt moet worden voor de overslag invoer. Wanneer dit niet wordt ingevuld moet dit later worden afgeleid uit bijv een shapefile met uittredepunten. Daarvoor zijn specifieke workflows beschikbaar. 	 |
-| bekledingen    	  | Optioneel 	| Moet overeen komen met kolom 'doorsnede' bij [Stabiliteit_default.csv](Macro.md). Hiermee wordt een link gemaakt tussen vak en doorsnede die gebruikt moet worden voor de stabiliteit invoer. Wanneer dit niet wordt ingevuld moet dit later worden afgeleid uit bijv een shapefile met uittredepunten. Daarvoor zijn specifieke workflows beschikbaar. 	                     |
+| piping          	 | Optioneel 	| Moet overeen komen met een waarde in de kolom `doorsnede` bij [Piping_default.csv](Piping.md). Hiermee wordt aangegeven welke doorsnede bij welk vak hoort. Let op: bij het maken van de vakindeling is deze invoer optioneel, maar bij het maken van een [database](Genereren_database.md) moeten alle doorsneden in het `geojson` bestand zijn ingevuld.                              |
```

Comment:
Volgorde van faalmechanismen komt hier niet overeen met csv. In Vakindeling csv eerst stabiliteit, dan piping, dan overslag, dan bekledingen, en daarnakunstwerken. 

Bekledingen mist nu ook in deze beschrijving en stabiliteit staat er dubbel in.